pipelines/tagger_parser_ud/configs/default.cfg

```diff
@@ -146,15 +174,16 @@ eps = 0.00000001
 learn_rate = 0.001
 
 [training.score_weights]
+tag_acc = 0.16
+pos_acc = 0.08
+morph_acc = 0.08
 morph_per_feat = null
+lemma_acc = 0.08
+dep_uas = 0.08
+dep_las = 0.08
 dep_las_per_type = null
 sents_p = null
 sents_r = null
-tag_acc = 0.33
-pos_acc = 0.17
-morph_acc = 0.17
-dep_uas = 0.17
-dep_las = 0.17
 sents_f = 0.0
```

Comment:
These don't sum up to 1 anymore, which feels nicer? Maybe we can just have each weighted as 0.16?